### PR TITLE
[Update] several tasks

### PR DESCRIPTION
**Pull request by hanyulo**
Done:
1. Two Types of TRLink (with style or without)
2. FadeIn Section 0.5 -> 1 
3. Remove First Module from FadeIn Animation
4. Remove Hover Effect from mobile device
5. Editor Pick Section
    At first scene, only display selected divs 

Working On:
Post Load for Image 
Lazy Load Image 
src - srcset

### CODE REVIEW

File: src/components/animations/fadein-fadeout.js [hunk 1]
```diff
@@ -11,17 +16,19 @@ class FadeInFadeOut extends React.Component {
...
+  componentWillReceiveProps(nextProps) {
+    // when user click on button and start animation to change the selected div.
+    for (const property in nextProps) {
+      if (Object.hasOwnProperty.call(this.props, property)) {
+        if (property === 'isSelected' && nextProps[property] !== this.props[property]) {
+          this.setState({
+            onAnimate: true,
+          })
+        }
+      }
```
Comment (nickhsine) [inline src/components/animations/fadein-fadeout.js]: I am not so sure about this.
Could we just code 
```
if (_.get(nextProps, 'isSelected') !== _.get(this.props, 'isSelected')) {
  this.setState({
    onAnimate: true 
  })
}
```

File: src/components/side-bar.js [hunk 2]
```diff
@@ -163,7 +163,9 @@ class SideBar extends React.Component {
   }
 
   _handleOnFadeIn(upComingSection) {
-    this.fadeInSectionMap[upComingSection].startAnimation()
+    if (upComingSection !== 'editorPick') {
```
Comment (nickhsine) [inline src/components/side-bar.js]: You should check the order of sections, not the name.
If we rename the `editorPick`, we need change this place as well.

File: src/components/side-bar.js [hunk 3]
```diff
@@ -189,14 +191,20 @@ class SideBar extends React.Component {
...
+                  :
+                  <div>
+                    <ScrollFadein
+                      ref={(node) => { this.fadeInSectionMap[moduleId] = node }}
+                      moduleId={moduleId}
+                    >
+                      {singleModule}
+                    </ScrollFadein>
+                  </div>
+                }
```
Comment (nickhsine) [inline src/components/side-bar.js]: Don't place the first section in the for loop.

File: src/components/animations/fadein-fadeout.js [hunk 4]
```diff
@@ -54,9 +64,7 @@ class FadeInFadeOut extends React.Component {
       >
         <div
           ref={(node) => { this.Node = node }}
-          style={{
-            display: 'none',
-          }}
+          style={style}
         >
           {this.props.children}
         </div>
```
Comment (nickhsine) [inline src/components/animations/fadein-fadeout.js]: Add `TODO` to mention this function should be refactored.

File: src/components/animations/fadein-fadeout.js (hunk 1 above)
Comment (hanyulo) [inline src/components/animations/fadein-fadeout.js]: 👍

File: src/components/animations/fadein-fadeout.js (hunk 4 above)
Comment (hanyulo) [inline src/components/animations/fadein-fadeout.js]: 👍

File: src/components/side-bar.js (hunk 2 above)
Comment (hanyulo) [inline src/components/side-bar.js]: 👍

File: src/components/side-bar.js (hunk 3 above)
Comment (hanyulo) [inline src/components/side-bar.js]: 👍

**Comment by nickhsine** [comment]
LGTM 👍 🔥 ♠️

**Pull request merged**